plejd/mqtt.js

```diff
@@ -2,7 +2,7 @@ const EventEmitter = require('events');
 const mqtt = require('mqtt');
 const _ = require('lodash');
 
-const startTopic = 'hass/status';
+const startTopic = 'homeassistant/status';
```

Comment:
Am I correct to assume that if the user has followed the instructions and setup his own birth_message as suggested "hass/status". Then Homeassistant won't fire this new default birth message?

This is then a breaking change. I would be a non-breaking-change if we listened to both topics, but I don't know if it's worth it?

Reply:
Oh, I'm sorry. I didn't read the instructions 🤦 

You're right.
`homeassistant/status` is the default value, though, so it might be worth listening for both and actually remove that part of the instructions in favor for setting up the MQTT integration in Home Assistant via the GUI...

I can update this PR tonight.